src/handlers/express.ts

```diff
@@ -56,6 +56,7 @@ export function expressRequestHandler(
         // We push the current correlationId out as a header so that a TrackJS
         // agent on the client-side of the request can link up with us.
         res.setHeader("TrackJS-Correlation-Id", correlationId);
+        res.setHeader("Access-Control-Expose-Headers", "TrackJS-Correlation-Id");
```

Comment:
Quick question - if they've already set Access-Control-Expose-Headers for some other purpose is this additive or does it tromp? (I can't remember how that works)